proto/path/qos/evm.proto

```diff
@@ -6,6 +6,16 @@ option go_package = "github.com/buildwithgrove/path/observation/qos";
 import "path/qos/jsonrpc.proto";
 import "path/metadata/metadata.proto";
 
+// EVMResponseInvalidReason defines why an endpoint response was rejected.
+// Current invalid response types (as of PR #164):
+//   1. EmptyResponse - endpoint returned no data
+//   2. UnmarshalErr - response failed to parse into expected format
+enum EVMResponseInvalidReason {
+ REASON_UNSPECIFIED = 0;
+ REASON_EMPTY_RESPONSE = 1; // No response data received
+ REASON_UNMARSHAL_ERR = 2;  // Response parsing failed
+}
```

Comment:
```suggestion
enum EVMResponseInvalidReason {
	EVM_RESPONSE_INVALID_REASON_UNSPECIFIED = 0;
	EVM_RESPONSE_INVALID_REASON_EMPTY = 1;      // No response data received
	EVM_RESPONSE_INVALID_REASON_UNMARSHAL = 2;  // Response parsing failed
}
```

The main reason I'm suggesting this:

<img width="653" alt="Screenshot 2025-02-13 at 9 10 58 PM" src="https://github.com/user-attachments/assets/0eda7d14-c314-4172-9e15-156f75e3311c" />